### PR TITLE
Reviewed with new guidelines for commiting changes.

### PR DESCRIPTION
**Pull request by ESapenaVentura**
### Release notes

For `docs/commiters.md`:
- Added pre-requisites
- Updated obsolete information about commiting process

For `docs/release_process.md`:
- Updated environments
- Updated outdated documentation

### Reviews requested

- Need wranglers to review as this is an SOP change

### CODE REVIEW

**Comment by ESapenaVentura** [comment]
Tagging @amnonkhen and @Wkt8 as reviewers

File: docs/committers.md [hunk 1]
```diff
@@ -60,39 +60,43 @@ This document serves as an SOP for committers who are ultimately responsible for
...
-1. **Switch** to the develop branch. All suggested changes should be based on the current develop branch.
+1. **Switch** to the staging branch. All suggested changes should be based on the current staging branch.
 
-        git checkout develop
+        git checkout staging
 
-1. **Make** and **switch** to a new working branch from develop. Name the branch following the convention: `initials-brief-desc-of-branch-scope`. Optionally, you can tag a GitHub issue (e.g. `Issue222`) or JIRA ticket (e.g. `HCA-123`) in the branch name.
+1. **Make** and **switch** to a new working branch from staging. Name the branch following the convention: `initials-brief-desc-of-branch-scope`. Optionally, you can tag a GitHub issue (e.g. `Issue222`) in the branch name.
 
         git checkout -b mf-new-mouse-module-Issue222
```
Comment (amnonkhen) [inline docs/committers.md:97]:
```suggestion
        git checkout -b mf-new-mouse-module-dcp-222
```

File: docs/committers.md [hunk 2]
```diff
@@ -60,39 +60,43 @@ This document serves as an SOP for committers who are ultimately responsible for
...
         cd metadata-schema
     
-1. **Switch** to the develop branch. All suggested changes should be based on the current develop branch.
+1. **Switch** to the staging branch. All suggested changes should be based on the current staging branch.
 
-        git checkout develop
+        git checkout staging
 
-1. **Make** and **switch** to a new working branch from develop. Name the branch following the convention: `initials-brief-desc-of-branch-scope`. Optionally, you can tag a GitHub issue (e.g. `Issue222`) or JIRA ticket (e.g. `HCA-123`) in the branch name.
+1. **Make** and **switch** to a new working branch from staging. Name the branch following the convention: `initials-brief-desc-of-branch-scope`. Optionally, you can tag a GitHub issue (e.g. `Issue222`) in the branch name.
```
Comment (amnonkhen) [inline docs/committers.md:95]:
```suggestion
1. **Make** and **switch** to a new working branch from staging. Name the branch following the convention: `initials-brief-desc-of-branch-scope`. Optionally, you can tag a GitHub issue (e.g. `dcp-222`) in the branch name.
```

**Review by amnonkhen — CHANGES_REQUESTED** [review]
Generally OK. A few minor comments.

File: docs/committers.md (hunk 2 above)
Comment (ESapenaVentura) [reply docs/committers.md:95]: Why this change? Is `Issue` as a preffix not clear?

File: docs/release_process.md [hunk 3]
```diff
@@ -253,7 +229,7 @@ c. develop
 
 1. Wait for travis tests to pass and **merge the PR**.
 
-1. **Repeat** for the next branch until it is hotfixed to `master`, `staging`, `integration` and `develop` environments.
+1. **Repeat** for the next branch until it is hotfixed to `master` and `staging` environments.
 
 After merging the hotfix to all release branches, **Delete the branch**.
```
Comment (Wkt8) [inline docs/release_process.md:235]: Would add here some steps for changing the spreadsheet template located in the geo_to_hca folder. 

1. In the [geo_to_hca ](https://github.com/ebi-ait/geo_to_hca/tree/master/docs) repo, download the latest version of the hca_template, and edit to add the new metadata schema changes. 
2. Move the now outdated template to the [outdated templates](https://github.com/ebi-ait/geo_to_hca/tree/master/docs/Outdated%20Templates) folder, and upload the new up-to-date template following DDMMYYhca_template.xlsx to the repo. 
3. Notify the other wranglers to download the new version of the spreadsheet template.

File: docs/release_process.md [hunk 4]
```diff
@@ -144,44 +143,27 @@ Anyone on the metadata team can trigger a primary release from develop to integr
 
 1. **Commit** your changes
 
-        git commit -a -m "Release from develop to integration YYYY-MM-DD."
+        git commit -a -m "Release from staging to master YYYY-MM-DD."
         git push origin develop
```
Comment (ESapenaVentura) [inline docs/release_process.md:147]:
```suggestion
        git push origin staging
```

File: docs/committers.md (hunk 2 above)
Comment (ESapenaVentura) [reply docs/committers.md:95]: @amnonkhen

**Pull request merged**